basis/compiler/cfg/builder/blocks/blocks.factor

```diff
@@ -35,11 +35,12 @@ IN: compiler.cfg.builder.blocks
 : call-height ( #call -- n )
     [ out-d>> length ] [ in-d>> length ] bi - ;
 
+: emit-call-block ( word height -- )
+    dup adjust-d ##call, make-kill-block ;
```

Comment:
Isn't this wrong, since its ``( word height -- )``, this calls ``adjust-d`` and ``##call,`` on height?  When it should instead be:

```[ ##call, ] [ adjust-d ] bi* make-kill-block```

?